src/components/AppDropdownItem.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+    <div>
+        <slot/>
+    </div>
+</template>
+
+<script>
+export default {
+    name: "AppDropdownItem"
+};
+</script>
```

Comment:
Please add a blank line at the end of file and configure your editor to automatically do.

This rule is repeated again and again and you seem still don't configure your editor?